### ISSUE TITLE
Accessibility Swarm on Radio Schedules component

### ISSUE THREAD

**Issue by tochwill** [bbc/simorgh#5236]
**Is your feature request related to a problem? Please describe.**
Carry out a team accessibility swarm for the Radio Schedules component once it is completed.

This could be done when integrating the alpha component locally in Simorgh.

Screen reader UX can be found here: https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5dc0724850731722a4aa767a

Accessibility criteria can be found here: https://docs.google.com/document/d/10rzotmSurDgpsZJgbdxvZ_HOzypJa-qHo5tCuoSr_t0/edit#heading=h.3icatbqe6ylx

**Describe the solution you'd like**
Go through the accessibility checklist and fix any issue we might face.

**Testing notes**
Screen Reader UX is within Zeplin designs.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by tochwill** [comment]
Blocked on code review and UX review for integrating radio schedules (@tochwill link to the story)

**Comment by DenisHdz** [comment]
Once we complete all the a11y bugs tickets we found during our team swarm, we should have a 2 person a11y swarm to confirm everything is fine.

- [Program Card styling bugs](https://github.com/bbc/psammead/issues/3300) ✅
- [Program Card styling bug on IE11 high contrast mode](https://github.com/bbc/psammead/issues/3310) ✅
- [Radio Schedule: Firefox high contrast border bug](https://github.com/bbc/psammead/issues/3323) ✅
- [Remove white border from Program Card](https://github.com/bbc/psammead/pull/3345) ✅
- [VoiceOver on Mac OS is not reading the start time on radio schedules within the title, but it is reading it out when interacting with the duration time element.](https://github.com/bbc/psammead/issues/3306) ❌
- [When using VoiceOver on text, it is read with an English accent. On a number, it is read out in English](https://github.com/bbc/psammead/issues/3307) ✅
- [Simplify the DOM structure within Headings in the radio schedule component ](https://github.com/bbc/psammead/issues/3305) ✅
- [Radio Schedules start time styling bugs](https://github.com/bbc/psammead/issues/3301) ✅
- [Radio Schedules - Duration markup](https://github.com/bbc/psammead/issues/3303) ✅
- [Radio Schedules - VoiceOver reading out wrongly the NEXT card headline](https://github.com/bbc/psammead/issues/3347)  ✅

**Comment by DenisHdz** [comment]
After having the second a11y swarm and fixed the [bug](https://docs.google.com/spreadsheets/d/1_Ajk8KXxD_azbrv7MiHEMvS-hDB3iM_mnwxlT09Rgd0/edit#gid=0) we found, this can be closed and we can bring the component out of alpha.
